assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -52,6 +52,7 @@
 	--color-link-dark: #{ $muriel-blue-700 };
 
 	--color-section-nav-item-background-hover: #{ $muriel-blue-0 };
+	--color-themes-active-text: #{ $muriel-blue-0 };
```

Comment:
at this point we probably want to add a generic CSS variable for `$muriel-blue-0`